lib/NetPacksLib.cpp

```diff
@@ -365,6 +365,19 @@ DLL_LINKAGE void PlayerEndsGame::applyGs(CGameState *gs)
 	}
 }
 
+DLL_LINKAGE void PlayerReinitInterface::applyGs(CGameState *gs)
+{
+	if(!gs || !gs->scenarioOps)
+		return;
+	
+	//TODO: what does mean if more that one player connected?

```

Comment:
As far as I understood each connection can have a few players in hot-seat mode

Reply:
that should work. The scenario described here is when several connections are related to the one player. I can't imagine such scenario but looking through the code it seems possible